src/go/migration-cli/cmd/root.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2023 Redpanda Data, Inc.
```

Comment:
can we name this directory `operator-v1-v2-migration` because migration-cli can be anything.